components/cluster/cluster-glue/cluster-glue.p5m

```diff
@@ -0,0 +1,231 @@
+file path=usr/lib/stonith/plugins/external/dracmc-telnet
+file path=usr/lib/stonith/plugins/external/hetzner
+file path=usr/lib/stonith/plugins/external/hmchttp
+file path=usr/lib/stonith/plugins/external/ibmrsa
+file path=usr/lib/stonith/plugins/external/ibmrsa-telnet
+file path=usr/lib/stonith/plugins/external/ipmi
+file path=usr/lib/stonith/plugins/external/ippower9258
+file path=usr/lib/stonith/plugins/external/kdumpcheck
+file path=usr/lib/stonith/plugins/external/libvirt
+file path=usr/lib/stonith/plugins/external/nut
+file path=usr/lib/stonith/plugins/external/rackpdu
+file path=usr/lib/stonith/plugins/external/riloe
+file path=usr/lib/stonith/plugins/external/ssh
+file path=usr/lib/stonith/plugins/external/vcenter
+file path=usr/lib/stonith/plugins/external/vmware
+file path=usr/lib/stonith/plugins/external/xen0
+file path=usr/lib/stonith/plugins/external/xen0-ha
+file path=usr/lib/stonith/plugins/stonith2/apcmaster.a
+file path=usr/lib/stonith/plugins/stonith2/apcmaster.so
+file path=usr/lib/stonith/plugins/stonith2/apcmastersnmp.a
+file path=usr/lib/stonith/plugins/stonith2/apcmastersnmp.so
+file path=usr/lib/stonith/plugins/stonith2/apcsmart.a
+file path=usr/lib/stonith/plugins/stonith2/apcsmart.so
+file path=usr/lib/stonith/plugins/stonith2/baytech.a
+file path=usr/lib/stonith/plugins/stonith2/baytech.so
+file path=usr/lib/stonith/plugins/stonith2/cyclades.a
+file path=usr/lib/stonith/plugins/stonith2/cyclades.so
+file path=usr/lib/stonith/plugins/stonith2/drac3.a
+file path=usr/lib/stonith/plugins/stonith2/drac3.so
+file path=usr/lib/stonith/plugins/stonith2/external.a
+file path=usr/lib/stonith/plugins/stonith2/external.so
+file path=usr/lib/stonith/plugins/stonith2/ibmhmc.a
+file path=usr/lib/stonith/plugins/stonith2/ibmhmc.so
+file path=usr/lib/stonith/plugins/stonith2/meatware.a
+file path=usr/lib/stonith/plugins/stonith2/meatware.so
+file path=usr/lib/stonith/plugins/stonith2/null.a
+file path=usr/lib/stonith/plugins/stonith2/null.so
+file path=usr/lib/stonith/plugins/stonith2/nw_rpc100s.a
+file path=usr/lib/stonith/plugins/stonith2/nw_rpc100s.so
+file path=usr/lib/stonith/plugins/stonith2/rcd_serial.a
+file path=usr/lib/stonith/plugins/stonith2/rcd_serial.so
+file path=usr/lib/stonith/plugins/stonith2/rhcs.a
+file path=usr/lib/stonith/plugins/stonith2/rhcs.so
+file path=usr/lib/stonith/plugins/stonith2/ribcl.py
+file path=usr/lib/stonith/plugins/stonith2/rps10.a
+file path=usr/lib/stonith/plugins/stonith2/rps10.so
+file path=usr/lib/stonith/plugins/stonith2/ssh.a
+file path=usr/lib/stonith/plugins/stonith2/ssh.so
+file path=usr/lib/stonith/plugins/stonith2/suicide.a
+file path=usr/lib/stonith/plugins/stonith2/suicide.so
+file path=usr/lib/stonith/plugins/stonith2/wti_mpc.a
+file path=usr/lib/stonith/plugins/stonith2/wti_mpc.so
+file path=usr/lib/stonith/plugins/stonith2/wti_nps.a
+file path=usr/lib/stonith/plugins/stonith2/wti_nps.so
+file path=usr/lib/stonith/plugins/xen0-ha-dom0-stonith-helper
```

Comment:
Should *.so and executable in /usr/lib/stonith/plugins/* be mode=0555? You can use a transform.